Misc/NEWS.d/next/Windows/2024-12-05-00-21-11.gh-issue-80192.28PTj4.rst

```diff
@@ -0,0 +1 @@
+Fixed valid :mod:`ssl` certificates being rejected.
```

Comment:
Agree with Andrew's comment - this needs to be significantly expanded, because this is a very impactful change.

We also would need a section in `Doc/whatsnew/3.14.rst` explaining the impact and options for workarounds.

We _might_ also need to use a deprecation period to turn this on by default, though part of me would prefer not to.